src/js/components/Pagination/Pagination.js

```diff
@@ -127,10 +142,8 @@ const Pagination = forwardRef(
         disabled: activePage === 1,
         icon: <PreviousIcon color={iconColor} />,
         onClick: event => {
-          event.persist();
-          const adjustedEvent = event;
-          adjustedEvent.page = activePage - 1;
-          handleClick(adjustedEvent);
+          const nextPage = activePage - 1;
+          handleClick(event, nextPage);
```

Comment:
This way we explicitly have `nextPage`, `previousPage`, and `page`

```suggestion
          const previousPage = activePage - 1;
          handleClick(event, previousPage);
```